src/ostree/ot-dump.c

```diff
@@ -251,11 +263,35 @@ ot_dump_summary_bytes (GBytes          *summary_bytes,
 
   g_variant_iter_init (&iter, exts);
 
-  /* XXX Should we print something more human-friendly for
-   *     known extension names like 'ostree.static-deltas'? */
   while (g_variant_iter_loop (&iter, "{sv}", &key, &value))
     {
-      g_autofree char *string = g_variant_print (value, FALSE);
-      g_print ("%s: %s\n", key, string);
+      g_autofree gchar *value_str = NULL;
+      const gchar *pretty_key = NULL;
+
+      if (g_strcmp0 (key, "ostree.static-deltas") == 0)
+        {
+          pretty_key = "Static Deltas";
+          value_str = g_variant_print (value, FALSE);
+        }
+      else if (g_strcmp0 (key, "ostree.summary.last-modified") == 0)
```

Comment:
I’ve pushed a follow-up commit on the branch which switches to using all the existing `#define`s.